deployment/CE01ISSM_Deploy.csv

```diff
@@ -24,9 +24,9 @@ deploymentNumber,Reference Designator,sensor.uid,startDateTime,stopDateTime,lat,
 1,CE01ISSM-SBC11-00-CPMENG000,CGCON-ECPM01-00002,2014-04-17T20:45:00,2014-08-16T22:30:00,44.65828,-124.0956,CGMCE-01ISSM-00001,OC1404B,,1,,,,,0,25,
 1,CE01ISSM-SBD17-00-DCLENG000,CGCON-EDCL17-00002,2014-04-17T20:45:00,2014-08-16T22:30:00,44.65828,-124.0956,CGMCE-01ISSM-00001,OC1404B,,1,,,,,0,25,
 1,CE01ISSM-SBD17-01-MOPAK0000,OL000200,2014-04-17T20:45:00,2014-08-16T22:30:00,44.65828,-124.0956,CGMCE-01ISSM-00001,OC1404B,,1,,,,,0,25,
-1,CE01ISSM-SBD17-04-VELPTA000,CGINS-VELPTA-08535,2014-04-17T20:45:00,2014-08-16T22:30:00,44.65828,-124.0956,CGMCE-01ISSM-00001,OC1404B,,1,,,,,1,25,
-1,CE01ISSM-SBD17-06-CTDBPC000,,2014-04-17T20:45:00,2014-08-16T22:30:00,44.65828,-124.0956,CGMCE-01ISSM-00001,#OC1404B,,1,,,,,1,25,Not Deployed
-1,CE01ISSM-SBD17-06-FLORTD000,,2014-04-17T20:45:00,2014-08-16T22:30:00,44.65828,-124.0956,CGMCE-01ISSM-00001,#OC1404B,,1,,,,,1,25,Not Deployed
+1,CE01ISSM-SBD17-04-VELPTA000,CGINS-VELPTA-08535,2014-04-17T20:45:00,2014-08-16T22:30:00,44.65828,-124.0956,CGMCE-01ISSM-00001,OC1404B,,1,,,,,1.3,25,
+1,CE01ISSM-SBD17-06-CTDBPC000,,2014-04-17T20:45:00,2014-08-16T22:30:00,44.65828,-124.0956,CGMCE-01ISSM-00001,#OC1404B,,1,,,,,0.9,25,Not Deployed
+1,CE01ISSM-SBD17-06-FLORTD000,,2014-04-17T20:45:00,2014-08-16T22:30:00,44.65828,-124.0956,CGMCE-01ISSM-00001,#OC1404B,,1,,,,,1.5,25,Not Deployed
```

Comment:
The FLORT is at the same depth as the VELPT on the ISSM moorings. Both values should reflect the nominal depth for the VELPT based on it's pressure record.